java-backend/src/main/java/org/kframework/backend/java/kil/TermContext.java

```diff
@@ -17,38 +15,32 @@
 /**
  * An object containing context specific to a particular configuration.
  */
-public class TermContext extends JavaSymbolicObject {
+public class TermContext {
```

Comment:
please explain why this is safe.

Reply:
Because this extension is wrong and not used anywhere. That's why this PR is a cleanup - removing unused stuff.